source/includes/gcp/_forwarding_rules.md

```diff
@@ -1,9 +1,150 @@
+      "networkTier": "PREMIUM",
+      "portRange": "80-80",
+      "selfLink": "https://www.googleapis.com/compute/v1/projects/test-area/global/forwardingRules/forwardingrulename",
+      "target": "https://www.googleapis.com/compute/v1/projects/test-area/global/targetHttpProxies/targetproxyname",
+      "shortTarget": "targetproxyname",
+      "id": "4724455712741277576",
+      "name": "gfr-user-name"
+   }
+}
+```
+
+<code>GET /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/forwardingrules/:id</code>
+
+Retrieve a forwarding rule in a given [environment](#administration-environments)
+
+Attributes | &nbsp;
+------- | -----------
+`creationTimestamp`<br/>*string* | Creation timestamp in RFC3339 text format.
+`description` <br/>*string* | An optional description of this resource.
+`IPAddress`<br/>*string* | The IP address that this forwarding rule serves. 
+`IPProtocol`<br/>*enum* | The IP protocol to which this rule applies. The default protocol is TCP.
+`ipVersion`<br/>*enum* | The IP version that will be used by this forwarding rule. The default IP version is IPV4.
+`kind`<br/>*string* | Type of the resource.
+`loadBalancingScheme`<br/>*enum* | Specifies the forwarding rule type. Possible values are INTERNAL and EXTERNAL.
+`networkTier`<br/>*enum* | This signifies the networking tier used for configuring this load balancer.
+`portRange`<br/>*string* | The destination ports that this forwarding rule forwards packets with.
+`selfLink`<br/>*string* | Server-defined URL for this resource.
+`target`<br/>*string* |  The URL of the target resource to receive the matched traffic.
+`shortTarget`<br/>*string* | The name of the target resource.
+`id`<br/>*string* | The unique identifier for the resource. This identifier is defined by the server.
+`name`<br/>*string* | Name of the resource. Provided by the client when the resource is created.
+
+<!-------------------- CREATE A FORWARDING RULE -------------------->
+##### Create a forwarding rule
+
+```shell
+curl -X POST \
+  -H 'MC-Api-Key: your_api_key' \
+  -H "Content-Type: application/json" \
+  -d "request _body" \
+  "https://cloudmc_endpoint/v1/services/gcp/test-area/forwardingrules"
+```
+
+<code>POST /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/forwardingrules</code>
+
+Create a new forwarding rule
+
+Required | &nbsp;
+------- | -----------
+`name`<br/>*string* | The display name of the forwarding rule
+`portRange`<br/>*string* | The destination ports that forwarding rule forwards packets with.
+`shortTarget`<br/>*string* | The name of the target resource.
+
+Optional | &nbsp;
+------- | -----------
+`description`<br/>*string* | Description of the forwarding rule.
+`reserveStatic`<br/>*boolean* | Whether the operation reserves a new static IP or not. The default value is false.
```

Comment:
Can we rename this variable, it's not clear what it is, `reserveStaticIp`

Reply:
need a bit more detail here, as the example JSON doesn't show all possible scenarios, which we could do.

If the value is false and if no `shortIPAddress` is provided, an ephemeral external IP address will be assigned. 
If the value is false and `shortIPAddress` is provided, then the existing external IP address will be assigned.
If the value is true, a new external static IP will be reserved and assigned.

Reply:
done